src/Controls/Internal/TextCopyPasteHelper.ts

```diff
@@ -0,0 +1,87 @@
+﻿module Fayde.Controls.Internal {
+    export class TextCopyPasteHelper {
+        private callback: any = false;
+        private pastedText: string = "";
+
+		GetPastedText(): string {
+            return this.pastedText;
+        }
+
+        private createElement(text: string): HTMLDivElement {
+            (<any>document.body).scroll = "no";
```

Comment:
If we modify `document.body`, we need to be aware of these changes in the Fayde application or surface.  I think we should be cautious of using this.